openfasoc/generators/temp-sense-gen/flow/design/sky130hd/tempsense/config.mk

```diff
@@ -46,3 +46,6 @@ export PRE_GLOBAL_ROUTE = $(SCRIPTS_DIR)/openfasoc/pre_global_route.tcl
 
 # informs any short circuits that should be forced during routing
 export CUSTOM_CONNECTION 	= ../blocks/$(PLATFORM)/tempsenseInst_custom_net.txt
+
+# indicates in how many points the VIN route from the HEADER cells connects to its power ring
```

Comment:
What do you mean?

Reply:
This has to do with the route from the header cells to the VIN voltage domain. It says that the value of the env variable below (VIN_ROUTE_CONNECTION_POINTS) tells the generator how many pins should be used to connect that route to the voltage domain. More pins = less resistance

Btw should I rephrase this?